recipes/libcurl/all/conanfile.py

```diff
@@ -54,6 +54,11 @@ class LibcurlConan(ConanFile):
     def _is_mingw(self):
         return self.settings.os == "Windows" and self.settings.compiler != "Visual Studio"
 
+    @property
+    def _is_win_x_android(self):
+        return self.settings.os == "Android" and self.settings.os_build == "Windows"
```

Comment:
@jgsogo Can we detect it using the new cross-building feature?

Reply:
just realize, maybe the `settings.os_build` is useless anyway for this purpose, 
`if platform.system() == "Windows":`  should tell me if I am on Windows
then I can remove the os_build from settings, and get rid of the warning ...

Reply:
so, Android build fails with configure on Windows? do you have an error message?

Reply:
o dam, sorry , I just messed up this branch, too tired, 

and yes, on windows this configure script can not be executed 

I will close this PR and make a new one ....